aten/src/THC/THCTensorRandom.cu

```diff
@@ -62,33 +53,17 @@ THC_API __host__ void THCRandom_setRNGState(THCState* state, THByteTensor *rng_s
     THArgCheck(THByteTensor_nElement(rng_state) == total_size, 1, "RNG state is wrong size");
   }
   THArgCheck(THByteTensor_isContiguous(rng_state), 1, "RNG state must be contiguous");
-  memcpy(&gen->state.initial_seed, THByteTensor_data(rng_state) + states_size, seed_size);
+  uint64_t input_seed = *((uint64_t*)(THByteTensor_data(rng_state) + states_size));
```

Comment:
FYI this technically violates strict aliasing. The original memcpy code is better (as it does not violate strict aliasing)

Reply:
(You can always memcpy into `uint64_t input_seed` and then use the public API)